synopsys-task/src/synopsys-task/input.ts

```diff
@@ -31,6 +31,20 @@ export function getInput(
   return "";
 }
 
+export function getArbitraryInputs(
```

Comment:
I think this method is used for handling multiple params for classic editor and yaml. But we are not returning classicEditorKey from this method. Will this work?

Reply:
addressed